packages/containers/package.json

```diff
@@ -141,6 +141,8 @@
   },
   "version": "0.131.0",
   "dependencies": {
+    "rc-slider": "8.4.1",
+    "react-ace": "5.8.0",
```

Comment:
Why do you add react-ace ?

Reply:
my storybook broke without it, am i the only one ?

Reply:
Storybook is just unhappy without react-ace -> 
```
WARNING in ./node_modules/@talend/react-forms/lib/widgets/CodeWidget/CodeWidget.component.js
Module not found: Error: Can't resolve 'react-ace' in '/pathtogithub/ui/packages/containers/node_modules/@talend/react-forms/lib/widgets/CodeWidget'
```
should i remove it as it "work" on storybook ?